repository-data/application/src/main/resources/hcm-content/taxonomies.yaml

```diff
@@ -0,0 +1,2836 @@
+/content/taxonomies:
+  jcr:primaryType: hippotaxonomy:container
+  /GlobalProfessions:
+    jcr:primaryType: hippo:handle
+    jcr:mixinTypes: ['hippo:versionInfo']
+    hippo:versionHistory: 724343aa_7729_4f98_a70d_6778dc4ee8db
+    /GlobalProfessions[1]:
+      jcr:primaryType: hippotaxonomy:taxonomy
+      jcr:mixinTypes: ['hippostd:publishableSummary', 'mix:referenceable', 'mix:versionable']
+      jcr:uuid: 2f46ac86-5518-4776-a0a3-b0a62f40d051
+      hippo:availability: [preview]
+      hippostd:retainable: false
+      hippostd:state: unpublished
+      hippostdpubwf:createdBy: admin
+      hippostdpubwf:creationDate: 2023-08-04T07:52:31.051+01:00
+      hippostdpubwf:lastModificationDate: 2023-08-04T07:58:08.177+01:00
+      hippostdpubwf:lastModifiedBy: admin
+      hippotaxonomy:defaultLocale: en
+      hippotaxonomy:locales: [en]
+      /advanced_critical_care_practitioners:
+        jcr:primaryType: hippotaxonomy:category
+        hippotaxonomy:key: advanced_critical_care_practitioners
+        locale.en.name: Advanced critical care practitioners
+        /anaesthesia_associate:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: anaesthesia_associate
+          locale.en.name: Anaesthesia associate
+        /physician_associate:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: physician_associate
+          locale.en.name: Physician associate
+        /surgical_care_practitioner:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: surgical_care_practitioner
+          locale.en.name: Surgical care practitioner
+      /allied_health_professionals:
+        jcr:primaryType: hippotaxonomy:category
+        hippotaxonomy:key: allied_health_professionals
+        locale.en.name: Allied health professionals
+        /art_therapist_x002f_art_psychotherapist:
```

Comment:
Can we fix the name and key ?

Reply:
Going through and changing all of these now